databricks_cli/runs/cli.py

```diff
@@ -52,10 +54,11 @@ def _runs_to_table(runs_json):
     for r in runs_json.get('runs', []):
         run_id = r.get('run_id', 'no_run_id')
         run_name = r.get('run_name', 'no_run_name')
-        life_cycle_state = r.get('state', {}).get('life_cycle_state', 'no_life_cycle_state')
-        result_state = r.get('state', {}).get('result_state', 'no_result_state')
-        run_page_url = r.get('run_page_url', 'no_run_page_url')
-        ret.append((run_id, run_name, life_cycle_state, result_state, run_page_url))
+        life_cycle_state = r.get('state', {}).get('life_cycle_state', 'n/a')
+        result_state = r.get('state', {}).get('result_state', 'n/a')
+        run_page_url = r.get('run_page_url', 'n/a')
```

Comment:
just realized that we don't output run page url for normal job runs. maybe we should return the run page url as well (not part of this PR)